x/ibc/applications/transfer/keeper/relay_test.go

```diff
@@ -28,28 +29,28 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 	}{
 		{"successful transfer from source chain",
 			func() {
-				_, _, connA, connB := suite.coordinator.SetupClientConnections(suite.chainA, suite.chainB, ibctesting.Tendermint)
```

Comment:
Hmm, you could have just changed `ibctesting.Tendermint = exported.Tendermint`. That's why I used the testing package so future changes would be easy